Specs/Number/Japanese/NumberModel.json

```diff
@@ -1319,8 +1317,8 @@
     ]
   },
   {
-    "Input": "21.2E0",
-    "Comment": "PendingValidation",
+    "Input": "21.2e0",
```

Comment:
Both should work. Please don't modify input unless it was really incorrect (i.e., would never happen in the real world).